Lib/test/test_json/test_detect_encoding.py

```diff
@@ -0,0 +1,38 @@
+import codecs
+
+from test.test_json import PyTest, CTest
+
+
+class TestDetectEncoding:
```

Comment:
Isn't it already tested in `test_bytes_decode`?

`detect_encoding` is not a part of the public API, it does not need direct tests.

Reply:
It is, I've removed it.